src/Stratis.Bitcoin.Features.Interop/EthereumClient/IEthereumClientBase.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Numerics;
+using Nethereum.Contracts;
+using Stratis.Bitcoin.Features.Interop.Models;
+
+namespace Stratis.Bitcoin.Features.Interop.EthereumClient
+{
+    public interface IEthereumClientBase
```

Comment:
@noescape00 @zeptin ah I see now you implemented this class... any reason why this method can't be made async and return a `Task`?